econml/dml.py

```diff
@@ -134,6 +134,26 @@ def effect(self, T0, T1, X):
             T1 = self._one_hot_encoder.transform(reshape(self._label_encoder.transform(T1), (-1, 1)))[:, 1:]
         return super().effect(T0, T1, X)
 
+    def score(self, Y, T, X, W=None):
```

Comment:
Also consider whether to include an example of its use in the notebook.

Reply:
I will create another issue on it.